src/main/resources/config/application.yml

```
@@ -100,4 +99,7 @@ rest:
     base-url: "\${WORKFLOW_HUB_BASE_URL:http://localhost:8080}"
     max-attempts: "\${WORKFLOW_HUB_MAX_ATTEMPTS:3}" # It's counting also the first attempt
     wait-time-millis: "\${WORKFLOW_HUB_WAIT_TIME_MILLIS:500}"
-    print-body-when-error: "\${WORKFLOW_HUB_PRINT_BODY_WHEN_ERROR:true}"
\ No newline at end of file
+    print-body-when-error: "\${WORKFLOW_HUB_PRINT_BODY_WHEN_ERROR:true}"
+
+debt-position-pagination:
+  page-size: "\${DEBT_POSITIONS_PAGE_SIZE:1000}"
```

let's structurate in order to allow for future pagination requirements
```suggestion
query-limits:
  default:
     size: "\${DEFAULT_PAGE_SIZE:1000}"
  debt-positions:
     size: "\${DEBT_POSITIONS_PAGE_SIZE:\${query-limits.default.size}}"
```